website/docs/d/account_groups.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "ibm"
+page_title: "IBM : account_groups"
```

Comment:
title should be enterprise_XXXX across all docs

Reply:
Done